.github/workflows/auto_cherry_pick.yml

```diff
@@ -35,6 +35,39 @@ jobs:
       - name: Cherry pick into ${{ matrix.to_branch }}
         uses: jyejare/github-cherry-pick-action@main
         with:
+          token: ${{ secrets.CHERRYPICK_PAT }}
           branch: ${{ matrix.to_branch }}
           labels: |
             Auto_Cherry_Picked
+            ${{ matrix.to_branch }}
+          assignees: |
+            ${{ github.event.pull_request.user.login }}
+
+  create-issue:
+    runs-on: ubuntu-latest
+    if: ${{ always() && contains(join(needs.*.result, ','), 'failure') }}
+    needs: auto-cherry-pick
+    steps:
+      - name: Create Issue on Failed Auto Cherrypick
+        uses: dacbd/create-issue-action@main
+        with:
+          token: ${{ secrets.CHERRYPICK_PAT }}
+          title: "[Failed-AutoCherryPick] - ${{ github.event.pull_request.title }}"
+          body: |
+            #### Auto-Cherry-Pick WorkFlow Failure:
+            - [Failed Cherrypick Action](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})
+            - [Parent Pull Request](https://github.com/${{ github.repository }}/pull/${{ github.event.number }})
+          labels: Failed_AutoCherryPick
+          assignees: ${{ github.event.pull_request.user.login }}
+
+  google-chat-notification:
+    runs-on: ubuntu-latest
+    if: ${{ always() && contains(join(needs.*.result, ','), 'failure') }}
+    needs: auto-cherry-pick
+    steps:
+      - name: Google Chat Notification - Airgun
+        uses: Co-qn/google-chat-notification@releases/v1
+        with:
+          name: "[Failed-AutoCherryPick] - ${{ github.event.pull_request.title }}"
```

Comment:
I think it would be better to have some diff here in naming of card notification
```suggestion
          name: "[${{ github.repository }}-AutoCherryPick-Failed] - ${{ github.event.pull_request.title }}"
```

Reply:
I would go further and filter this with the repo name with a title and skip `AutoCherryPick-Failed` because the bot name itself suggests that it is auto cherry-pick fail.

Reply:
Not required for repository name as the repository name appears in notification :

<img width="515" alt="Screenshot 2022-11-03 at 12 43 44 PM" src="https://user-images.githubusercontent.com/11752425/199665518-abd4a64f-48cf-4f6f-99dc-bba5079b37c1.png">


But I can remove `AutoCherryPick-Failed` from title as its a duplication.

Reply:
Agree